docs/framework/deployment/index.md

```diff
@@ -28,7 +28,7 @@ ms.author: "mairaw"
 manager: "wpickett"
 ---
 # Deploying the .NET Framework and Applications
-This article helps you get started deploying the .NET Framework with your application. Most of the information is intended for developers, OEMs, and enterprise administrators. Users who want to install the .NET Framework on their computers should read [Installing the .NET Framework 4.5](../../../docs/framework/getting-started/install-the-net-framework.md).  
+This article helps you get started deploying the .NET Framework with your application. Most of the information is intended for developers, OEMs, and enterprise administrators. Users who want to install the .NET Framework on their computers should read [Installing the .NET Framework](~/docs/framework/install/guide-for-developers.md).  
```

Comment:
I took that from the getting started with the .NET Framework doc:  /docs/framework/get-started/index.md; and since docs/framework/deployment/index seems like a dev topic, it seemed like the dev install topic would be good. I'll change it now.